### PR TITLE
Fix sketch tool `gr.Image` not filling up the entire component size

### PR DESCRIPTION
**Pull request by space-nuko**
# Description

Closes: #3534

2023-03-27 18_47_04-Gradio - Chromium

I'm not sure what the `.fixed-height` class was meant to achieve, can @pngwn or someone else clarify?

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have added a short summary of my change to the CHANGELOG.md
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by abidlabs** [comment]
Thanks @space-nuko! As you may have noticed, we just [moved all of the frontend files](https://github.com/gradio-app/gradio/pull/3605/files). Maybe not the best timing given all of your PRs, but if you could kindly update your PRs to reflect the changes in the new files, we'll do our best to review the PRs quickly.

**Comment by pngwn** [comment]
Pretty sure we don't want to remove this entirely. Will take a closer look at some point but might take me a while to get to.

Image is a low priority atm because it will be rewritten shortly and generally works ok. I'd like to minimise the amount of time I spend on this until we have changed some things.

**Comment by space-nuko** [comment]
Okay thanks for the update @pngwn. Is there a tracking issue for such rewrite yet?

**Comment by pngwn** [comment]
It's here: https://github.com/gradio-app/gradio/issues/466

I've started work on this in the background so this is very much on the radar and coming when a few other things are taken care off. As a result we want to minimise effort spent on the Image component, outside of simple fixes because we don't want to waste effort on something that will be removed shortly.

File: js/atoms/src/Block.svelte [hunk 1]
```diff
@@ -32,6 +32,7 @@
 		"" +
 		(typeof style.height === "number" ? `height: ${style.height}px; ` : "") +
 		(typeof style.width === "number" ? `width: ${style.width}px;` : "");
+	console.log(style);
```
Comment (abidlabs) [inline js/atoms/src/Block.svelte:35]: Remove

Comment (abidlabs) [inline js/atoms/src/Block.svelte:35]:
```suggestion
```

**Comment by abidlabs** [comment]
@aliabid94 can confirm the original issue is fixed. However, the cropper (which I think you were also trying to fix in this PR) is behaving very strangely. Here's a GIF:

```py
import gradio as gr

gr.Interface(
    lambda x: x,
    gr.Image(source='image', tool='editor'),
    [gr.Image()],
).launch()
```

Recording 2023-04-10 at 14 03 29

**Review by abidlabs — APPROVED** [review]

**Comment by abidlabs** [comment]
Thanks @space-nuko and @aliabid94 for the fix!

**Pull request merged**